### PR TITLE
GH9: Added basic build script

### PR DESCRIPTION
**Pull request by gep13**
Based on the main Cake Build Script.

Doesn't do anything yet, apart from generate the Release Notes using GitReleaseManager.

### CODE REVIEW

**Comment by gep13** [comment]
@patriksvensson can you have a quick scan over this? Thanks!

**Comment by patriksvensson** [comment]
@gep13 Sure thing

File: build/parameters.cake [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+                password: context.EnvironmentVariable("CAKEVSO_GITHUB_PASSWORD")
+            ),
+            IsPublishBuild = new [] {
+                "ReleaseNotes",
+                "Create-Release-Notes"
+            }.Any(
+                releaseTarget => StringComparer.OrdinalIgnoreCase.Equals(releaseTarget, target)
+            ),
+            IsReleaseBuild = new [] {
+                "Publish",
```
Comment (patriksvensson) [inline build/parameters.cake]: Should these targets be here?

File: build/parameters.cake [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+                "Publish-NuGet",
+                "Publish-Chocolatey",
```
Comment (devlead) [inline build/parameters.cake]: these can probably be cleaned out

**Comment by patriksvensson** [comment]
@gep13 Looks good to me. Left a minor comment.

File: build/parameters.cake (hunk 1 above)
Comment (gep13) [inline build/parameters.cake]: @patriksvensson @devlead good catch! I did go through and clear out a lot of stuff, but missed this :+1:

**Comment by gep13** [comment]
@patriksvensson fixed up and rebased.

**Comment by patriksvensson** [comment]
@gep13 Perfect. Will merge.

**Pull request merged**